src/features/checker/types/evaluation.ts

```diff
@@ -52,8 +52,8 @@ export interface EvaluationBody {
   alloPoolId: string;
   alloApplicationId: string;
   cid: string;
-  evaluator: string;
+  evaluator?: string;
   summaryInput: EvaluationSummaryInput;
   evaluationStatus: EvaluationStatus;
-  signature: Hex;
+  signature?: Hex;
```

Comment:
Is there a scenario where this can be optional ?

Reply:
Yes, I made it that way because the signature is passed from the wrapper app.

If it weren’t set up like this, we would need to pass the signature as "0x" within the Gitcoin UI and then modify it in the wrapper app.